test/indexer/test_indexer_controller.py

```diff
@@ -0,0 +1,150 @@
+import json
+import logging
+
+import boto3
+from chalice.app import (
+    SQSRecord,
+)
+import mock
+from moto import (
+    mock_sqs,
+    mock_sts,
+)
+
+from azul import (
+    config,
+    queues,
+)
+from azul.azulclient import (
+    AzulClient,
+)
+from azul.indexer import (
+    BundleFQID,
+)
+from azul.indexer.index_controller import (
+    IndexController,
+)
+from azul.logging import (
+    configure_test_logging,
+)
+from indexer import (
+    ForcedRefreshIndexService,
+    IndexerTestCase,
+)
+
+logger = logging.getLogger(__name__)
+
+
+# noinspection PyPep8Naming
+def setUpModule():
+    configure_test_logging(logger)
+
+
+class IndexControllerTestCase(IndexerTestCase):
+
+    def setUp(self) -> None:
+        super().setUp()
+        self.index_service.create_indices(self.catalog)
+        self.client = AzulClient()
+        self.controller = IndexController()
+        self.queue_manager = queues.Queues(delete=True)
+
+    def tearDown(self):
+        self.index_service.delete_indices(self.catalog)
+        super().tearDown()
+
+    def generate_events(self):
+        pass
+
+    def _create_mock_queues(self):
```

Comment:
Is there a reason why the decorator would not be operational? I was running into this issue when trying to add @mock_sqs and @mock_sts to the `_create_mock_queues` definition and it would never mock the sts functionality correctly, it would try to using an invalid token to perform the creation of the mock queues.

I do agree that the positioning of the `@mock_x` would be better if it was closer to the method definition, i was forced to follow precedence from another file, https://github.com/DataBiosphere/azul/blob/e0b7febd31003f3555a322ca4354bdca18b14056/test/health_check_test_case.py#L282 which also lead me to the naming decision, but also does the mocking the same way.

Reply:
I like the idea of wrapping the class, and not having to add decorators everywhere. 👍🏽  got it working thx.